.github/workflows/deploy-orch.yml

```diff
@@ -0,0 +1,57 @@
+name: Deploy Orchestration
+
+env:
+  AWS_REGION: eu-west-2
+
+#gha role is build Auth github actions role in Gds dev Account
+#Artifact bucket is Build Auth deploy github artifcat Source bucket
+#Destination Bucket is auth-lambda-signed bucket from prod tooling account
+
+on:
+  workflow_run:
+    workflows: ["Build modules"]
+    types:
+      - completed
+
+jobs:
+  deploy:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: gradle
+
+      - name: Set up Gradle
+        uses: gradle/gradle-build-action@v2
```

Comment:
What do we need these for?

Reply:
@ethanmills isn't this required for SAM build/deploy?
Its used in other github deployments:
[life events](https://github.com/govuk-one-login/life-events-platform/blob/main/.github/workflows/application-main.yml)
[demo app](https://github.com/govuk-one-login/devplatform-demo-sam-app/blob/main/.github/workflows/sam-app-deploy-using-environment-secrets.yml)

Reply:
Those also build the Java, here it looks like we're outsourcing that to `Build modules`

Reply:
Ahh yep different pipeline structure 👍